package.json

```diff
@@ -80,32 +80,31 @@
     "docker-toolbelt": "^3.3.7",
     "duration-js": "^4.0.0",
     "event-stream": "3.3.4",
-    "express": "^4.0.0",
+    "express": "^4.17.1",
     "fork-ts-checker-webpack-plugin": "^0.5.2",
-    "fp-ts": "^2.0.0",
-    "husky": "^4.2.1",
+    "fp-ts": "^2.0.5",
```

Comment:
This is not actually updated, just setting the package.json version to match the package-lock.json